server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
Perhaps I'm mis-understanding? From our end we're just `npm install`-ing - then bundling the result into a package - we don't ever use hobknob as a package itself within `node_modules`?